apps/condo/domains/news/schema/NewsItemRecipientsExportTask.test.js

```diff
@@ -113,12 +113,14 @@ describe('NewsItemRecipientsExportTask', () => {
             test('admin can', async () => {
                 const [objCreated] = await createTestNewsItemRecipientsExportTask(adminClient, adminClient.user, dummyO10n)
 
-                const [obj, attrs] = await updateTestNewsItemRecipientsExportTask(adminClient, objCreated.id)
+                await waitFor(async () => {
+                    const [obj, attrs] = await updateTestNewsItemRecipientsExportTask(adminClient, objCreated.id)
 
-                expect(obj.dv).toEqual(1)
-                expect(obj.sender).toEqual(attrs.sender)
-                expect(obj.v).toEqual(2)
-                expect(obj.updatedBy).toEqual(expect.objectContaining({ id: adminClient.user.id }))
+                    expect(obj.dv).toEqual(1)
+                    expect(obj.sender).toEqual(attrs.sender)
+                    expect(obj.v).toEqual(3)
+                    expect(obj.updatedBy).toEqual(expect.objectContaining({ id: adminClient.user.id }))
```

Comment:
Are fields `sender` and `updatedBy same for `v=2` and `v=3` ?

Reply:
done